backend/src/auth/oidc.strategy.ts

```
@@ -0,0 +1,31 @@
+import { Issuer, Strategy as OIDCStrategy } from 'openid-client';
+
+export async function createOidcStrategy() {
+  const issuerUrl = process.env.OIDC_ISSUER!;
```

@Drilio 

I just see the ! at the end of config reading. Have you done it in purpose ?

If consts are not set, it can fail silently. I suggest to add something like : 
```js
if (!issuerUrl || !clientId || !clientSecret || !redirectUri) {
  throw new Error('Missing required OIDC environment variables');
}
```

Note : I have forget a bit how the constants works, so perhaps my remark is dumb in that case